cmd/migrate_status.go

```diff
@@ -20,6 +21,7 @@ func NewMigrateStatusCmd(slOpts []servicelocatorx.Option, dOpts []driver.Options
 		RunE:  cli.NewHandler(slOpts, dOpts, cOpts).Migration.MigrateStatus,
 	}
 
+	cmdx.RegisterFormatFlags(cmd.PersistentFlags())
```

Comment:
Why are these needed? Not that it would break anything, but if they are not actually supported, it is probably confusing for users when the flags show up in the help messages.

Reply:
These are in fact required and the program will abort if they are not registered.

Reply:
This was previously a bug which I worked around in cloud. This fixes the core issue.